src/index.js

```diff
@@ -0,0 +1,127 @@
+    const button4Element = document.createElement("button")
+    button4Element.id = "button-4"
+    button4Element.innerText = "4"
+    const button5Element = document.createElement("button")
+    button5Element.id = "button-5"
+    button5Element.innerText = "5"
+    const button6Element = document.createElement("button")
+    button6Element.id = "button-6"
+    button6Element.innerText = "6"
+    const thirdLineWrapper = document.createElement("div")
+    thirdLineWrapper.className = "line-wrapper"
+    const button7Element = document.createElement("button")
+    button7Element.id = "button-7"
+    button7Element.innerText = "7"
+    const button8Element = document.createElement("button")
+    button8Element.id = "button-8"
+    button8Element.innerText = "8"
+    const button9Element = document.createElement("button")
+    button9Element.id = "button-9"
+    button9Element.innerText = "9"
+    const fourthLineWrapper = document.createElement("div")
+    fourthLineWrapper.className = "line-wrapper"
+    const buttonStarElement = document.createElement("button")
+    buttonStarElement.id = "star"
+    buttonStarElement.innerText = "*"
+    const button0Element = document.createElement("button")
+    button0Element.id = "button-0"
+    button0Element.innerText = "0"
+    const buttonHashElement = document.createElement("button")
+    buttonHashElement.id = "hash"
+    buttonHashElement.innerText = "#"
+    const fifthLineWrapper = document.createElement("div")
+    fifthLineWrapper.className = "line-wrapper"
+    const buttonCallElement = document.createElement("button")
+    buttonCallElement.id = "call"
+    buttonCallElement.innerText = "Call"
+    const buttonXElement = document.createElement("button")
+    buttonXElement.id = "button-x"
+    buttonXElement.innerText = "X"
+
+    headerElement.appendChild(mainElement)
+    mainElement.appendChild(buttonWrapperElement)
+    buttonWrapperElement.appendChild(firstLineWrapper)
+    firstLineWrapper.appendChild(button1Element)
+    firstLineWrapper.appendChild(button2Element)
+    firstLineWrapper.appendChild(button3Element)
+    buttonWrapperElement.appendChild(secondLineWrapper)
+    secondLineWrapper.appendChild(button4Element)
+    secondLineWrapper.appendChild(button5Element)
+    secondLineWrapper.appendChild(button6Element)
+    buttonWrapperElement.appendChild(thirdLineWrapper)
+    thirdLineWrapper.appendChild(button7Element)
+    thirdLineWrapper.appendChild(button8Element)
+    thirdLineWrapper.appendChild(button9Element)
+    buttonWrapperElement.appendChild(fourthLineWrapper)
+    fourthLineWrapper.appendChild(buttonStarElement)
+    fourthLineWrapper.appendChild(button0Element)
+    fourthLineWrapper.appendChild(buttonHashElement)
+    buttonWrapperElement.appendChild(fifthLineWrapper)
+    fifthLineWrapper.appendChild(buttonCallElement)
+    fifthLineWrapper.appendChild(buttonXElement)
+
+    // FOOTER
+
+    const footerElement = document.createElement("footer")
```

Comment:
Same here, what is there would be milion buttons?

create array with buttons config and then use relevant lambda function to handle elements creation.